### PR TITLE
Correcting issue with <>s

### PR DESCRIPTION
**Pull request by gesslar**
* Removed triangular  brackets from `language-configuration.json`
* Removed `activationEvents` from `package.json`
* Updated `README.md`

Please test this PR before accepting. I was having some strange issues with .c files with `'` quotes in comments having weird effects on subsequent syntax highlighting. But only sometimes. It was really bizarre.

### CODE REVIEW

**Comment by gesslar** [comment]
This apparently doesn't seem to fix the issue and I think it might have to do with the fact that there are no operators in `syntaxes/lpc.tmLanguage`. 🤷🏻

**Comment by gesslar** [comment]
For reference on operators in the language file https://github.com/atom/language-c/blob/master/grammars/c.cson

File: syntaxes/lpc.tmLanguage [hunk 1]
```diff
@@ -958,6 +986,79 @@
...
+					</dict>
+					<dict>
+						<key>match</key>
+						<string>&amp;=|\^=|&lt;&lt;=|&gt;&gt;=|\|=</string>
+						<key>name</key>
+						<string>keyword.operator.assignment.compound.bitwise.lpc</string>
+					</dict>
+					<dict>
+						<key>match</key>
+						<string><<|>></string>
```
Comment (GwennKoi) [inline syntaxes/lpc.tmLanguage]: swap the < and > for &lt; and &gt; to not break the xml

Comment (GwennKoi) [reply syntaxes/lpc.tmLanguage]: aaaaannnnndddd it changed my text. (ampersand)lt; and (ampersand)gt;

File: README.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ LPmud C Language syntax highlighting for the FLUFFOS branch of LPmud drivers.
...
+
+### v1.1.1
+* Removed triangular brackets that were causing issues. `<` and `>` are not
+  used in FluffOS as brackets. If needed for other drivers, can re-evaluate
+  in the future.
+* Modified standard colour tag support to be arrays instead of objects, since
+  they are basic and don't require additional syntaxes such as "notIn".
+* Removed `activationEvents` from package.json since VS Code advises that this
+  is inferred information.
+
```
Comment (michaelprograms) [inline README.md:23]: This blurb should be added to `CHANGELOG.md` file also.

File: syntaxes/lpc.tmLanguage (hunk 1 above)
Comment (gesslar) [reply syntaxes/lpc.tmLanguage]: Yeah, I have that already, it's just not committed yet. ☺️

**Comment by gesslar** [comment]
This is ready for final review and merging if satisfactory.

**Review by GwennKoi — APPROVED** [review]
Thanks!

**Pull request merged**